app/models/game.rb

```diff
@@ -1,10 +1,10 @@
 # frozen_string_literal: true
 
 class Game < ApplicationRecord
-  has_one :creating_user, :class_name => 'User'
-  has_one :invited_user, :class_name => 'User'
-  has_one :first_move, :class_name => 'User'
-  has_one :winner, :class_name => 'User'
+  belongs_to :creating_user, :class_name => 'User', :foreign_key => 'creating_user_id'
+  belongs_to :invited_user, :class_name => 'User', :foreign_key => 'invited_user_id'
+  belongs_to :first_move, :class_name => 'User', :foreign_key => 'first_move_id'
+  belongs_to :winner, :class_name => 'User', :foreign_key => 'winner_id'
```

Comment:
You can not use `belongs_to :user` when associating one table with another multiple times. If we tried to use `belongs_to :user` rails would be confused which column you want to associate since none are named `user_id`.  Even if you renamed one to `user_id` you still would not be able to link the other columns to the User table. 
The way to associate multiple columns to one table is through foreign keys.

Reply:
https://stackoverflow.com/questions/49604547/have-multiple-users-belong-to-one-record-in-rails-5-app
This is a fairly decent summary of what is going on. Not actually the articles I used to help me but can't seem to find them right now.

Reply:
Okay, Added an optional tag to the three ID's that are not needed on game creation. Figured out how to use associations on FactoryBot to get the tests to pass without all the extra logic I had inserted. And reworked the `games_controller.rb` so that the link now works on the nav bar for the current user to create a game. Shane if you get a chance since this is your thread can you review the changes and resolve the conversation and I will merge it if it all looks good.

Reply:
All looks good now. Merging.